cmd/gpu_plugin/rm/gpu_plugin_resource_manager.go

```diff
@@ -416,6 +673,81 @@ func containerCards(pod *v1.Pod, gpuUsingContainerIndex int) []string {
 	return nil
 }
 
+func convertTileInfoToEnvMask(tileInfo string) string {
+	cardTileList := strings.Split(tileInfo, ",")
+
+	var tileIndices []string
+
+	for i, cardList := range cardTileList {
+		cards := strings.Split(cardList, ",")
```

Comment:
Is it really so that `cardTileList`'s elements may contain commas after what happens in the line 677?

Reply:
No, it's a curious case of unnecessary string splitting. I'll remove the second portion.